google-cloud-clients/google-cloud-asset/pom.xml

```diff
@@ -101,4 +88,4 @@
       </plugin>
     </plugins>
   </reporting>
-</project>
+</project>
```

Comment:
Here as well

Reply:
already add empty new line just below the last line.